### ISSUE TITLE
reportPrices reports NA for fegat in transport

### ISSUE THREAD

**Issue by orichters** [pik-piam/remind2#405]
With
```
curr_fe: fegat
```
[this line](https://github.com/pik-piam/remind2/blob/master/R/reportPrices.R#L396) and later lines report such wrong variables:
```
q <- as.quitte("NGFS_v4/remind-2023-04-12/output/C_SSP2-Base-rem-6/REMIND_generic_C_SSP2-Base-rem-6.mif")
grep("\\|NA", levels(q$variable), value = TRUE)

 [1] "Price|Marginal|Final Energy|Transport|ESR|NA"
 [2] "Price|Marginal|Final Energy|Transport|ESR|NA|Biomass"
 [3] "Price|Marginal|Final Energy|Transport|ESR|NA|Biomass|Moving Avg"
 [4] "Price|Marginal|Final Energy|Transport|ESR|NA|Efuel"
 [5] "Price|Marginal|Final Energy|Transport|ESR|NA|Efuel|Moving Avg"
 [6] "Price|Marginal|Final Energy|Transport|ESR|NA|Fossil"
 [7] "Price|Marginal|Final Energy|Transport|ESR|NA|Fossil|Moving Avg"
 [8] "Price|Marginal|Final Energy|Transport|ESR|NA|Moving Avg"
 [9] "Price|Marginal|Final Energy|Transport|NA"
[10] "Price|Marginal|Final Energy|Transport|NA|Biomass"
[11] "Price|Marginal|Final Energy|Transport|NA|Biomass|Moving Avg"
[12] "Price|Marginal|Final Energy|Transport|NA|Efuel"
[13] "Price|Marginal|Final Energy|Transport|NA|Efuel|Moving Avg"
[14] "Price|Marginal|Final Energy|Transport|NA|Fossil"
[15] "Price|Marginal|Final Energy|Transport|NA|Fossil|Moving Avg"
[16] "Price|Marginal|Final Energy|Transport|NA|Moving Avg"
[17] "Price|Marginal|Final Energy|Transport|other|NA"
[18] "Price|Marginal|Final Energy|Transport|other|NA|Biomass"
[19] "Price|Marginal|Final Energy|Transport|other|NA|Biomass|Moving Avg"
[20] "Price|Marginal|Final Energy|Transport|other|NA|Efuel"
[21] "Price|Marginal|Final Energy|Transport|other|NA|Efuel|Moving Avg"
[22] "Price|Marginal|Final Energy|Transport|other|NA|Fossil"
[23] "Price|Marginal|Final Energy|Transport|other|NA|Fossil|Moving Avg"
[24] "Price|Marginal|Final Energy|Transport|other|NA|Moving Avg"
```
My suspicion was that in [these lines](https://github.com/pik-piam/remind2/blob/master/R/reportPrices.R#L340-L351), `fegat = "Gases"` is missing, but since for example `Price|Marginal|Final Energy|Transport|Gases` is already reported (by `fegas`?), this may lead to duplicates, I'm not sure :)

So maybe you have a look first.

**Comment by orichters** [comment]
All right, I discuss it with Renato…

**Comment by johannah-pik** [comment]
Sorry, that I cannot really help out here.. hope you find a solution fast! And let us know if the root of the problem seems to be in the transport sector

**Comment by orichters** [comment]
We will fix that in #402 by adding fegat to the list there…
